doc/ref_cert/RC2/chapters/chapter04.md

```diff
@@ -3,3 +3,38 @@
 # 4. CNF Test Cases and Requirements Traceability
 
 ## Table of Contents
+* [4.1 Introduction](#4.1)
+* [4.2 Selection Criteria](#4.2)
+* [4.3 Traceability Matrix](#4.3)
+  * [4.3.1 Test Case Traceability to CNF Interoperability Requirements](#4.3.1)
```

Comment:
"CNF Interoperability" may be confusing since RA2 doesn't specify any CNF interoperability (between CNFs)? The aspects of CNFs to test are for portability i.e. CNF can run on different infrastructures

Reply:
from #1496 "Clear position agreed between CNTT and CNCF: RA2 shoudl only deal with requirements that workloads have in order to interoperate with an RA2-compliant platform. Cloud native best practices and the requirements relating to those will be handled in either the CNCF TUG or CNF Working Group."

Reply:
I would say it is interoperability between the CNF and the platform, not between CNFs. Does that make sense? Or is there a different way you would phrase it?

Reply:
It makes sense if you already know the details of position agreed between CNTT vs CNCF. There has been lots of confusion about that so we should try to clarify where possible. Either les make it explicit or perhaps "CNF portability" may be more appropriate?